app/api/config.py

```diff
@@ -0,0 +1,31 @@
+"""Configuration environment variables for the API."""
+
+from pydantic import Field, computed_field
+from pydantic_settings import BaseSettings
+
+
+class Settings(BaseSettings):
+    """Data model for configurable API settings."""
+
+    # NOTE: Environment variables are case-insensitive by default
+    # (see https://docs.pydantic.dev/latest/concepts/pydantic_settings/#case-sensitivity)
+    root_path: str = Field(alias="NB_NAPI_BASE_PATH", default="")
```

Comment:
Since it's `root_path` "inside" but `base_path` "outside", how about we make use of the `description` field of https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.Field to add a nice reminder / description of what this guy does